pkg/controller/compliancesuite/compliancesuite_controller.go

```diff
@@ -474,6 +476,14 @@ func (r *ReconcileComplianceSuite) reconcileRemediations(suite *compv1alpha1.Com
 		return reconcile.Result{}, err
 	}
 
+	// We only post-process when everything is done.
+	// This is to prevent unabled to unpause the MachineConfigPool
+	// when there are stucked scans.
+	if suite.Status.Phase != compv1alpha1.PhaseDone {
+		logger.Info("Waiting until all scans are in Done phase before post-procesing remediations")
```

Comment:
nit: post-processing*

But it looks like that typo existed prior to this patch, so we can also fix it separately.